### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by chloelu29**
- fixed tests import path in general 
- fixed database import path in endpoint testing 
- fixed export files in puppeteer

### CODE REVIEW

File: public/electron.jsx [hunk 1]
```diff
@@ -101,11 +101,14 @@ ipcMain.on('Universal.readFile', (e, filePath) => {
 });
 
 ipcMain.on('Universal.path', (e, folderPath, filePath) => {
+    console.log('folderPath:', folderPath)
+    console.log('filePath:', filePath)
```
Comment (Ruzeb) [inline public/electron.jsx:105]: remove console logs

File: src/components/TestMenu/EndpointTestMenu.tsx [hunk 2]
```diff
@@ -50,8 +50,7 @@ const EndpointTestMenu = () => {
   };
 
   const fileHandle = () => {
-    console.log('endpointTestCase'); 
-    console.log(endpointTestCase)
+    console.log('endpointTestCase:', endpointTestCase);
```
Comment (Ruzeb) [inline src/components/TestMenu/EndpointTestMenu.tsx:53]: remove

File: src/context/useGenerateTest.jsx [hunk 3]
```diff
@@ -403,13 +404,17 @@ function useGenerateTest(test, projectFilePath) {
     function createPathToReducers(statement) {
       let filePath = null;
       if (statement.reducersFilePath) {
-        let filePath = ipcRenderer.sendSync(
+        filePath = ipcRenderer.sendSync(
           'Universal.path',
           projectFilePath,
           statement.reducersFilePath
         );
+        console.log('filepath before replace', filePath)
```
Comment (Ruzeb) [inline src/context/useGenerateTest.jsx:412]: remove

File: src/context/useGenerateTest.jsx [hunk 4]
```diff
@@ -403,13 +404,17 @@ function useGenerateTest(test, projectFilePath) {
...
         filePath = filePath.replace(/\\/g, '/');
       }
+
+      console.log('filepath after replace:', filePath)
+
```
Comment (Ruzeb) [inline src/context/useGenerateTest.jsx:417]: remove

**Review by Ruzeb — CHANGES_REQUESTED** [review]

**Review by Ruzeb — APPROVED** [review]
lgtm

File: src/context/useGenerateTest.jsx [hunk 5]
```diff
@@ -470,19 +474,11 @@ function useGenerateTest(test, projectFilePath) {
...
-      //   hooksArr.push(hook);
-      // });
-      // let hookImports = hooksArr.reduce((str, curr) => {
-      //   str += `${curr}, `;
-      //   return str;
-      // }, '');
-
-      // const { hooksStatements } = hooksTestCase;
-      // const hooksStatements = hooksTestCase.hooksStatements;
+      console.log('createPathtoHooks statement:', statement);
```
Comment (alsdk850) [inline src/context/useGenerateTest.jsx:477]: delete!

File: src/context/useGenerateTest.jsx [hunk 6]
```diff
@@ -470,19 +474,11 @@ function useGenerateTest(test, projectFilePath) {
...
+  
 
       if (Array.isArray(hooksTestCase)) {
+        console.log('hooksTestCase:', hooksTestCase)
```
Comment (alsdk850) [inline src/context/useGenerateTest.jsx:481]: delete console!

File: src/context/useGenerateTest.jsx [hunk 7]
```diff
@@ -498,9 +494,14 @@ function useGenerateTest(test, projectFilePath) {
 
           testFileCode += `import { ${hookImports} } from '../${filePath}';`;
         }
+
+        
+
       } else if (typeof hooksTestCase === 'object') {
+        console.log('in hooksTestCase OBJECT:', hooksTestCase)
```
Comment (alsdk850) [inline src/context/useGenerateTest.jsx:501]: delete console

File: src/context/useGenerateTest.jsx (hunk 5 above)
Comment (alsdk850) [inline src/context/useGenerateTest.jsx:477]: s

**Review by alsdk850 — CHANGES_REQUESTED** [review]

**Review by alsdk850 — APPROVED** [review]

**Pull request merged**